content/hi/transport-layer-security.md

```diff
@@ -0,0 +1,29 @@
+---
+title: ट्रांसपोर्ट लेयर सिक्योरिटी (TLS)
+status: पूर्ण
```

Comment:
the value for status should not be localized. 
only valid english status is acceptable. I am sure the work in this PR is not visible.

Reply:
I have already added some missing words from my side around 2 weeks ago commit (https://github.com/cncf/glossary/pull/2593/commits/866d77e30d713c6d4749067a890230b8d93f9cf9). However, the status localization was skipped by me by mistake. Now this PR is pending for merge. If any other helpful suggestion is encountered by me or by anyone then I will update it